client/src/components/Signup.js

```
@@ -796,9 +810,9 @@ export default class Signup extends React.Component {
                             type="email"
                             required={true}
                             style={fieldStyle}
-                            error={!this.state.emailValid}
+                            error={!this.state.emailValid && this.state.email !== ''}
                         >
-                            {!this.state.emailValid ? getErrorLabel('Please enter a valid email!') : null}
+                            {!this.state.emailValid && this.state.email !== '' ? getErrorLabel('Please enter a valid email!') : null}
```

hides the error label when email is empty but other fields have been filled out